### PR TITLE
fix build error and warnings

### PR DESCRIPTION
**Pull request by sunghan-chang**

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [a423b6f488b1a0b36866536d1962a5c4562ff5c8] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [a423b6f488b1a0b36866536d1962a5c4562ff5c8] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [11e73e63d3b843c8885cc8b4618ed9e9bc47091d] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [11e73e63d3b843c8885cc8b4618ed9e9bc47091d] - Code Rule Check (C++) OK.

File: build/configs/artik053/cxxtest/Make.defs [hunk 1]
```diff
@@ -71,13 +71,15 @@ ifeq ($(WINTOOL),y)
...
   ARCHINCLUDES += -isystem "${shell cygpath -w $(TOPDIR)/net/lwip/src/include}"
   ARCHXXINCLUDES = -I. -isystem "${shell cygpath -w $(TOPDIR)/include}" -isystem "${shell cygpath -w $(TOPDIR)/../framework/include}" -isystem "${shell cygpath -w $(TOPDIR)/../external/include}" -isystem "${shell cygpath -w $(TOPDIR)/../external/include/libcxx}"
+  ARCHXXINCLUDES += -isystem "${shell cygpath -w $(TOPDIR)/net/lwip/src/include}"
   ARCHSCRIPT = -T "${shell cygpath -w $(TOPDIR)/../build/configs/artik053/scripts/$(LDSCRIPT)}"
 else
   # Linux/Cygwin-native toolchain
   MKDEP = $(TOPDIR)/tools/mkdeps$(HOSTEXEEXT)
   ARCHINCLUDES = -I. -isystem $(TOPDIR)/include -isystem $(TOPDIR)/../framework/include -isystem $(TOPDIR)/../external/include
-  ARCHINCLUDES += -isystem $(TOPDIR)/net/lwip/src/include/
+  ARCHINCLUDES += -isystem $(TOPDIR)/net/lwip/src/include
```
Comment (btheosam) [inline build/configs/artik053/cxxtest/Make.defs]: Don't this need -I flag?

**Comment by sunghan-chang** [comment]
> Don't ARCHXXXINCLUDE need -I flag?

It uses `-isystem` instead of -I. In this case, `-I` is not required.

**Pull request merged**